addons/web/static/src/views/calendar/calendar_common/calendar_common_popover.scss

```diff
@@ -75,13 +75,27 @@ $o-cw-popup-avatar-size: 16px;
     }
 }
 
-@for $i from 1 through length($o-colors-complete) {
-    $color: nth($o-colors-complete, $i);
-    $color-subtle: mix($o-white, $color, 55%);
+// avoid temporary values in global scope
```

Comment:
Looks quite clean to me, maybe just clarify the comment hereabove, and thanks for the live explanation :) would be nice to have a quick look from the web team